packages/client/components/AdhocTeamMultiSelect/AdhocTeamMultiSelect.tsx

```diff
@@ -0,0 +1,150 @@
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import Autocomplete from '@mui/material/Autocomplete'
+import TextField from '@mui/material/TextField'
+
+import {AdhocTeamMultiSelect_viewer$key} from '../../__generated__/AdhocTeamMultiSelect_viewer.graphql'
+import {Send as SendIcon} from '@mui/icons-material'
+import {Chip} from '../../ui/Chip/Chip'
+
+export type Option = {
+  id: string | null
+  label: string
+  email: string
+  picture: string | null
+  organizationIds: string[]
+}
+
+interface Props {
+  viewerRef: AdhocTeamMultiSelect_viewer$key
+  value: Option[]
+  onChange: (values: Option[]) => void
+}
+
+const autocompleteEmail = (input: string, domain: string) => {
+  if (input.endsWith(`@${domain}`) || input === '@') {
+    return input
+  }
+  const parts = input.split('@')
+
+  if (parts.length > 1 && (parts[1] ?? '').length > 0) {
```

Comment:
+1 this is a bit confusing to me. Could you add a comment and perhaps change the name `parts` to something more descriptive?